org.photoqt.PreviewQt.yml

```diff
@@ -0,0 +1,311 @@
+
+    ########################
+    # AVIF SUPPORT
+    ########################
+    - name: libavif
+      buildsystem: cmake-ninja
+      config-opts:
+        - -DCMAKE_BUILD_TYPE=Release
+        - -DAVIF_CODEC_AOM=ON
+        - -DAVIF_CODEC_DAV1D=ON
+        - -DAVIF_CODEC_AOM=ON
+      sources:
+        - type: git
+          url: https://github.com/AOMediaCodec/libavif
+          tag: v1.0.4
+          commit: d77cfece0b3e233ee4bed49dfef1e7dc451599f8
+
+    ########################
+    # JXL SUPPORT
+    ########################
+    - name: libjxl
+      buildsystem: cmake-ninja
+      config-opts:
+        - -DCMAKE_BUILD_TYPE=Release
+        - -DBUILD_TESTING=OFF
+      sources:
+        - type: git
+          url: https://github.com/libjxl/libjxl
+          tag: v0.10.2
+          commit: e1489592a770b989303b0edc5cc1dc447bbe0515
+
+    ########################
+    # EXR SUPPORT
+    ########################
+    - name: libdeflate
+      buildsystem: cmake-ninja
+      config-opts:
+        - -DCMAKE_BUILD_TYPE=Release
+      sources:
+        - type: git
+          url: https://github.com/ebiggers/libdeflate
+          tag: v1.19
+          commit: dd12ff2b36d603dbb7fa8838fe7e7176fcbd4f6f
+    ########################
+    - name: imath
+      buildsystem: cmake-ninja
+      config-opts:
+        - -DCMAKE_BUILD_TYPE=Release
+      sources:
+        - type: git
+          url: https://github.com/AcademySoftwareFoundation/Imath
+          tag: v3.1.11
+          commit: 5ac1d5335cb34f0f356c5f2461a57c845b96b115
+    ########################
+    - name: openexr
+      buildsystem: cmake-ninja
+      config-opts:
+        - -DCMAKE_BUILD_TYPE=Release
+      sources:
+        - type: git
+          url: https://github.com/AcademySoftwareFoundation/openexr
+          tag: v3.2.3
+          commit: e4b8d0226ad88f58b19936c79b9998c2f73d2cfa
+
+    ########################
+    # POPPLER DOCUMENTS
+    ########################
+    - name: poppler-data
+      buildsystem: cmake-ninja
+      builddir: true
+      config-opts:
+        - -DCMAKE_BUILD_TYPE=Release
+      sources:
+        - type: git
+          url: https://gitlab.freedesktop.org/poppler/poppler-data
+          tag: POPPLER_DATA_0_4_12
+          commit: af9f452b427d5ce8abe81ff98cc40c5f39fad90b
+    ########################
+    - name: boost
+      buildsystem: simple
+      build-commands:
+        - ./bootstrap.sh
+        - ./b2 install --prefix="$FLATPAK_DEST"
+      sources:
+        - type: archive
+          url: https://archives.boost.io/release/1.84.0/source/boost_1_84_0.tar.gz
+          sha256: a5800f405508f5df8114558ca9855d2640a2de8f0445f051fa1c7c3383045724
+    ########################
+    - name: openjpeg
+      buildsystem: cmake-ninja
+      config-opts:
+        - -DCMAKE_BUILD_TYPE=Release
+        - -DBUILD_STATIC_LIBS=OFF
+      sources:
+        - type: git
+          url: https://github.com/uclouvain/openjpeg
+          tag: v2.5.2
```

Comment:
The version in the runtime isn't enough?

```
$ pkg-config --modversion libopenjp2
2.5.0
```

Reply:
Thanks for pointing this out, removed it from the manifest.